package.json

```diff
@@ -85,6 +85,7 @@
     "@octokit/core": "^3.4.0",
     "@rollup/plugin-babel": "^5.2.0",
     "@rollup/plugin-node-resolve": "^13.0.4",
+    "@types/react-native": "^0.67.7",
```

Comment:
Just the latest version. Potentially it could make sense to sync with a version we're using -- but OTOH, I imagine a good fraction of changes to these type definitions are likely to be things that are also true of older versions, and I'd guess nobody generally puts in the work to backport those.

In any case I don't expect our results to be too sensitive to this version. Remember this isn't input we translate -- just context for TS to interpret the type definitions elsewhere that we do translate.